jobs/JGDAS_ENKF_DIAG

```diff
@@ -22,31 +22,33 @@ GDATE=$(${NDATE} -${assim_freq} ${CDATE})
 gPDY=$(echo ${GDATE} | cut -c1-8)
 gcyc=$(echo ${GDATE} | cut -c9-10)
 GDUMP=${GDUMP:-"gdas"}
```

Comment:
```suggestion
GDUMP="gdas"
```

Reply:
This wasn't something I changed, but I've changed it now to what you suggested (I had wondered the same thing).